frontend/public/components/utils/resource-log.tsx

```diff
@@ -402,6 +402,12 @@ export const ResourceLog: React.FC<ResourceLogProps> = ({
     true,
   );
 
+  React.useEffect(() => {
+    if (resource?.metadata?.annotations?.['console.openshift.io'] === 'wrap-log-lines') {
+      setWrapLines(true);
```

Comment:
Will this update the user's preference to wrap? I think we want to leave the preference as-is and only default to something else if the annotation is present.

Reply:
That's exactly what it does.  And it poses the UX concerns @sg00dwin noted above.  But it seems like it will be equally weird to change the wrap but possibly not have the control correctly indicate the current state (e.g., wrap lines is disabled by default and visiting a pod with the annotation results in wrap lines being enabled, but checkbox still shows disabled).

Reply:
Yeah if we do it this way, we want the control to indicate the current state. So we probably need to track the state of the control separate from the user preference value.